test/extensions/coin-machine.js

```diff
@@ -109,24 +109,20 @@ contract("Coin Machine", (accounts) => {
 
     it("can install the extension with the extension manager", async () => {
       ({ colony } = await setupRandomColony(colonyNetwork));
-      await colony.installExtension(COIN_MACHINE, coinMachineVersion, { from: USER0 });
-
-      await checkErrorRevert(
-        colony.installExtension(COIN_MACHINE, coinMachineVersion, { from: USER0 }),
-        "colony-network-extension-already-installed"
-      );
+      const tx = await colony.installExtension(COIN_MACHINE, coinMachineVersion, { from: USER0 });
 
-      await checkErrorRevert(colony.uninstallExtension(COIN_MACHINE, { from: USER1 }), "ds-auth-unauthorized");
+      const coinMachineAddress = getExtensionAddressFromTx(tx);
+      await checkErrorRevert(colony.methods["uninstallExtension(address)"](coinMachineAddress, { from: USER1 }), "ds-auth-unauthorized");
```

Comment:
If we're changing this overhead, how about a 'reinstall coinmachine' helper just in this test file? And/or look in to using `beforeEach`?